charts/spark-operator-chart/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: spark-operator
 description: A Helm chart for Spark on Kubernetes operator
-version: 1.2.11
-appVersion: v1beta2-1.4.3-3.5.0
+version: 1.2.12
+appVersion: v1beta2-1.4.3-3.5.1
```

Comment:
AppVersion should be `v1beta2-1.4.4-3.5.0`.   Here `1.4.3` is the version of the Spark Operator API and `3.5.0` is the version of Spark.

Reply:
i agree that this condition is not needed. it existed before i made changes there

Reply:
Ok, I've removed the offending action in this PR and updated the `appVersion`. Should be good now if either of you can review again 🙏